pkg/lib/serverTime.js

```diff
@@ -554,18 +552,19 @@ function ChangeSystimeBody({ state, errors, change }) {
     );
 
     const mode_options = [
-        <SelectOption key="manual_time" value="manual_time">{_("Manually")}</SelectOption>,
-        <SelectOption key="ntp_time" value="ntp_time" isDisabled={!ntp_supported}>{_("Automatically using NTP")}</SelectOption>
+        { value: "manual_time", content: _("Manually") },
+        { value: "ntp_time", content: _("Automatically using NTP") },
```

Comment:
This lost the isDisabled. It ought to be possible to specify that right here, as you do that below for ntp_time_custom

Reply:
Ouch, thanks!

Reply:
Fixed.